### ISSUE TITLE
Test failure with daylight saving times

### ISSUE THREAD

**Issue by gracinet** [alisaifee/hiro#2]
Hi,

I noticed the following test failure under Python 3.5:

```
======================================================================
FAIL: test_utc (tests.test_context_mgr.TestScaledContext)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/testenv/hiro/tests/test_context_mgr.py", line 41, in test_utc
    math.ceil((start_local - start_utc).seconds / 60.0 / 60.0))
AssertionError: 1 != 2
```

If I understand correctly, the test checks that the ``Timeline`` context manager does not introduce inconsistent timezone offsets.

My current timezone is Europe / Paris, and therefore changes from CET to CEST in spring.
I tried forcing the timezone to UTC, or af fixed offset, like UTC+0300, and the test passes.

That's why I think it's related to daylight saving time, and it may not be a false positive.

**Comment by alisaifee** [comment]
Thanks @gracinet - I'm able to reproduce this locally.

**Comment by alisaifee** [comment]
The problem was an incorrect use of `time.gmtime` to get utc.

**Comment by alisaifee** [comment]
Fix is in `0.3`
